smart/map_processing.py

```diff
@@ -0,0 +1,160 @@
+import numpy as np
+import skimage as ski
+from matplotlib import colors
+from skimage.morphology import disk
+
+import astropy.units as u
+
+from sunpy.map import Map, all_coordinates_from_map, coordinate_is_on_solar_disk
+
+
+def map_threshold(im_map):
+    """
+    Creating a map from a fits file and processing said map.
+
+    Parameters
+    ----------
+    file : Map
+        Unprocessed magnetogram map.
+
+    Returns
+    -------
+    im_map : Map
+        Processed magnetogram map.
+
+    """
+    im_map.data[~coordinate_is_on_solar_disk(all_coordinates_from_map(im_map))] = np.nan
+    im_map.cmap.set_bad("k")
+    im_map.plot_settings["norm"] = colors.Normalize(vmin=-200, vmax=200)
+    return im_map
+
+
+@u.quantity_input
+def STL(
```

Comment:
Similar to above re the name and maybe `smooth_thresshold_los` also looking at the code the order is smooth, los, threshold no?